MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -13,25 +13,34 @@ protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ errorModel: FeedErrorModel)
+}
+
 final class FeedPresenter {
 	private let feedView: FeedView
 	private let loadingView: FeedLoadingView
+	private let errorView: FeedErrorView
 	
-	init(feedView: FeedView, loadingView: FeedLoadingView) {
+	init(feedView: FeedView, loadingView: FeedLoadingView, errorView: FeedErrorView) {
 		self.feedView = feedView
 		self.loadingView = loadingView
+		self.errorView = errorView
 	}
 	
 	func didStartLoadingFeed() {
+		errorView.display(FeedErrorModel(errorText: .none))
 		loadingView.display(FeedLoadingViewModel(isLoading: true))
 	}
 	
 	func didFinishLoadingFeed(with feed: [FeedImage]) {
+		errorView.display(FeedErrorModel(errorText: .none))
 		feedView.display(FeedViewModel(feed: feed))
 		loadingView.display(FeedLoadingViewModel(isLoading: false))
 	}
 	
-	func didFinishLoadingFeed(with error: Error) {
+	func didFinishLoadingFeed(with errorMessage: String) {
+		errorView.display(FeedErrorModel(errorText: errorMessage))
```

Comment:
The presenter should decide which error message to display. It should receive an `Error` and then decide the message string.

```suggestion
	func didFinishLoadingFeed(with error: Error) {
		let errorText = "FEED_VIEW_CONNECTION_ERROR".localizedString()
		errorView.display(FeedErrorModel(errorText: errorText))
```